packages/addons/browser/chrome/source/resources/settings.xml

```diff
@@ -12,5 +12,7 @@
         <setting id="CUST_AUDIODEVICE_STR" type="text" label="30008" visible="eq(-1,true)" subsetting="true" default="" />
         <setting id="HIDE_CURSOR" type="bool" label="30009" default="false" />
         <setting id="DARK_MODE" type="bool" label="30010" default="false" />
+        <setting id="USE_CUST_USERAGENT" type="bool" label="30011" default="false" />
+        <setting id="CUST_USERAGENT_STR" type="text" label="30012" visible="eq(-1,true)" subsetting="true" default="" />
```

Comment:
Please insert a copy of the default user agent here and add a `sed` line in package.mk. Without an empty default user-agent is shown.
 
```suggestion
        <setting id="CUST_USERAGENT_STR" type="text" label="30012" visible="eq(-1,true)" subsetting="true" default="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/@CHROME_VERSION@" />
```

Reply:
done